terminology/constants.bal

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# This is a map of implemented search params for CodeSystems.
+# These are defined as a map because to make the search process ease.
+public const map<string> CODESYSTEMS_SEARCH_PARAMS = {
```

Comment:
Can we use enums for this purpose

Reply:
No. There are some mapping on it, so I think using ENUM won't solve that problem